web3/_utils/validation.py

```diff
@@ -152,10 +155,23 @@ def validate_abi_value(abi_type: TypeStr, value: Any) -> None:
     )
 
 
+def is_ordinary_string(value: Any) -> bool:
+    return (is_string(value) and not is_bytes(value) and not
```

Comment:
I think that the `is_string` check and the `is_bytes` check are mutually exclusive so you should be able to drop the `is_bytes` check.